src/main/java/org/broadinstitute/hellbender/tools/diagnostics/PrintBGZFBlockInformation.java

```diff
@@ -0,0 +1,189 @@
+package org.broadinstitute.hellbender.tools.diagnostics;
+
+import htsjdk.samtools.util.BlockCompressedStreamConstants;
+import htsjdk.tribble.AbstractFeatureReader;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.barclay.argparser.Argument;
+import org.broadinstitute.barclay.argparser.CommandLineProgramProperties;
+import org.broadinstitute.barclay.argparser.ExperimentalFeature;
+import org.broadinstitute.hellbender.cmdline.CommandLineProgram;
+import org.broadinstitute.hellbender.cmdline.StandardArgumentDefinitions;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.utils.io.IOUtils;
+import picard.cmdline.programgroups.DiagnosticsAndQCProgramGroup;
+
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Path;
+
+/**
+ * A diagnostic tool that prints information about the compressed blocks in a BGZF format file,
+ * such as a .vcf.gz file.
+ *
+ * The output looks like this:
```

Comment:
For properly formatted in the docgen, this requires the `<p>` HTML tag (I guess).

Reply:
Done